packages/react-router-dom/.size-snapshot.json

```diff
@@ -1,26 +1,26 @@
 {
```

Comment:
Let's leave this file out of the PR. It's just a build artifact that we should probably only generate when we cut releases.

Reply:
Hey. my bad, didn't see that sneak in.